src/main/app-updater.ts

```diff
@@ -53,10 +53,14 @@ export function startUpdateChecking(interval = 1000 * 60 * 60 * 24): void {
        * didn't ask for.
        */
       autoUpdater.autoInstallOnAppQuit = false;
-      installVersion = args.version;
+      installVersion = info.version;
 
+      autoUpdater.downloadUpdate()
```

Comment:
Do you still need to call `autoUpdater.downloadUpdate()` in `handleAutoUpdateBackChannel()`? (2 places)

Reply:
No, though it does check to see if the version has already been downloaded. I'll fix it in this PR though.